src/utils/three-utils.js

```diff
@@ -37,14 +37,8 @@ export function disposeNode(node) {
   }
 
   if (node.material) {
-    let materialArray;
-    if (node.material instanceof THREE.MeshFaceMaterial || node.material instanceof THREE.MultiMaterial) {
```

Comment:
Didn't see it in the migration guide, but dug in and looks like MeshFaceMaterial was removed so this makes sense. I don't think we were using it for anything anyway.